h2o-py/h2o/tree/tree.py

```diff
@@ -29,37 +29,113 @@ def __init__(self, model, tree_number, tree_class=None):
 
```

Comment:
Missing this change at the top:

Also, the description needs to change from:

"Represents a model of a Tree built by one of H2O’s algorithms (GBM, Random Forest)."
to
"Represents a model of a Tree built by one of H2O’s algorithms (GBM, Random Forest, XGBoost, Isolation Forest)."